src/plugins/validation/2and3/semantic-validators/info.js

```diff
@@ -0,0 +1,33 @@
+// Assertation 1:
+// check if info exists
+
+// Assertation 2:
+// making sure that the required version and title are defined properly
+module.exports.validate = function({ jsSpec }) {
+  const errors = [];
+  const warnings = [];
+  const no_input = !jsSpec.info;
```

Comment:
If an api definition has `info: 'abc'`, this validation will pass. We should check that it is an object. Also, whenever I think of edge cases that could've broken my code, I add tests for them. This helps ensure code quality and helps me think of better tests the next time I'm writing them. It would be a good idea to add this test in there 👍